samples/manager/manager.js

```diff
@@ -199,27 +199,33 @@ const createDevice = async (
     // optional auth parameters.
   });
 
-  const regPath = iotClient.registryPath(projectId, cloudRegion, registryId);
-  const device = {
-    id: deviceId,
-    credentials: [
-      {
-        publicKey: {
-          format: publicKeyFormat,
-          key: fs.readFileSync(publicKeyFile).toString(),
+  async function createDevice() {
+    // Construct request
+    const regPath = iotClient.registryPath(projectId, cloudRegion, registryId);
+    const device = {
+      id: deviceId,
+      credentials: [
+        {
+          publicKey: {
+            format: publicKeyFormat,
+            key: fs.readFileSync(publicKeyFile).toString(),
```

Comment:
We really try to avoid using `sync` methods in nodejs, because you would never use this in production.  Could I trouble you to use either `fs.promises`, or doing a `util.promisify(fs.readFile)`?

Reply:
Ok I will change it to `util.promisify(fs.readFile)`

Reply:
@bcoe The reason why I changed to async.